Makefile.am

```diff
@@ -202,6 +202,21 @@ ci: rc-release
 	@sudo rm -f $(ROOT_SITE_PACKAGES)/usercustomize.py
 	$(MAKE) coverage-report
 
+test-gui:
+	@rm -f tests/test-suite.log
+	@rm -rf tests/autogui-results-*/
+	@sudo mkdir -p $(ROOT_SITE_PACKAGES)
+	@sudo cp $(abs_builddir)/tests/usercustomize.py $(ROOT_SITE_PACKAGES)
+# We require SELinux in Permissive mode, see rhbz#1276376
+	@sudo setenforce 0
+	@sudo COVERAGE_PROCESS_START=$(abs_builddir)/.coveragerc \
+		$(abs_builddir)/tests/run_gui_tests.sh >tests/test-suite.log 2>&1
+	@sudo setenforce 1
```

Comment:
I don't think you should worry about calling setenforce here and on line 211.  First off, we're already testing for it with that decorator on DogtailTestCase.  Second, if you just unconditionally "setenforce 1" here, you'll be enabling it on systems that might have been set to permissive in the first place.